geomstats/vectorization_utils.py

```diff
@@ -0,0 +1,12 @@
+"""
```

Comment:
not a huge fan on the "utils" naming scheme. Sounds a bit like laziness to me when I see something with a utils suffix.